sdk/metric/exemplar/fixed_size_reservoir.go

```diff
@@ -53,9 +54,11 @@ type FixedSizeReservoir struct {
 }
 
 func newFixedSizeReservoir(s *storage) *FixedSizeReservoir {
+	var seed [32]byte
+	crand.Read(seed[:]) //nolint // crypto/rand never errors
 	r := &FixedSizeReservoir{
 		storage: s,
-		rng:     rand.New(rand.NewSource(time.Now().UnixNano())),
+		rng:     rand.New(rand.NewChaCha8(seed)),
```

Comment:
Since you are now using the free functions in the `rand` package below and no longer using `rng`, you could just remove this field and skip the now-unnecessary read from `crypto/rand`.

Reply:
ah right i forgot i did that